python/ray/workflow/workflow_storage.py

```diff
@@ -264,75 +265,99 @@ def load_step_output(self, step_id: StepID) -> Any:
         # In this case, there is no such step
         raise output_err
 
+    def save_dag(self, creator_task_id: StepID, state: WorkflowExecutionState) -> None:
```

Comment:
better call `save_workflow_execution_state` ? I actually expect a DAG object to be there with this name.